### DISCUSSION TITLE
Issue with Moving Window.

### DISCUSSION THREAD

**Discussion by rashid611** [epochpic/epoch#676] [General]
Hi,

I am currently using a moving window in my EPOCH simulations with the following settings:

```
begin:window
  move_window = T
  window_v_x = window_vx
  window_start_time = las_focus_time
  bc_x_min_after_move = simple_outflow
  bc_x_max_after_move = simple_outflow
end:window
```

Where `window_vx` is the same as group velxoity of laser.

However, I am facing an issue where the size of the number density plot in Visit reduces over time as the laser propagates. I prefer the plot size to remain constant throughout the simulation.

Could someone help me understand why this happens and how I can fix it ? 
![1](https://github.com/Warwick-Plasma/epoch/assets/143154854/905fbb45-4772-46ef-8763-542c0bee10b3)
![2](https://github.com/Warwick-Plasma/epoch/assets/143154854/d52481b8-f1d1-4682-bef6-30ecc46e9376)
![3](https://github.com/Warwick-Plasma/epoch/assets/143154854/49a28c53-931d-4ed8-a2a6-acad453d61e5)
Any suggestions or guidance would be greatly appreciated.

Thank you!

**Comment by TomGoffrey** [comment]
Your domain size looks to be about 60 microns, which is what is displayed in each plot. I would suggest consulting the VisIt documentation/support in setting the axis limits properly.

If you do think this is an issue with EPOCH please re-open the issue with more details.
